taglib/flac/flacfile.cpp

```diff
@@ -519,6 +519,16 @@ void FLAC::File::scan()
     const bool isLastBlock = (header[0] & LastBlockFlag) != 0;
     const unsigned int blockLength = header.toUInt(1U, 3U);
 
+    if (blockType >= MetadataBlock::Picture) {
+      nextBlockOffset += blockLength + 4;
```

Comment:
Please, document where this magic number comes from, and most importantly what is the meaning of it.

Reply:
Done.